bioblend/_tests/TestGalaxyObjects.py

```diff
@@ -112,7 +111,6 @@ def gi_module(self):
 
 
 class TestWrapper(unittest.TestCase):
-
```

Comment:
PEP 8 [states](https://www.python.org/dev/peps/pep-0008/#blank-lines) that all method definitions should surrounded by a single blank line. Can you revert this and the other blank line removals before the first class methods?

Reply:
Fixed by https://github.com/galaxyproject/bioblend/pull/247/commits/2ea309274d1d165fad357a1e62f2d19495745da5